addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -757,14 +769,16 @@ var SnippetsMenu = Widget.extend({
         this.window = this.ownerDocument.defaultView;
         this.$window = $(this.window);
 
+        // Create and initialize the theme customization menu widget.
+        this.themeCustomizationMenu = new ThemeCustomizationMenu(this, {tab: false});
+        defs.push(this.themeCustomizationMenu.appendTo($('<div>')));
         // Fetch snippet templates and compute it
-
         defs.push(this.loadSnippets().then(html => {
             return this._computeSnippetTemplates(html);
         }).then(() => {
-            this.customizePanel = document.createElement('div');
```

Comment:
I'd prefer this to stay an HTMLElement instead of a jQuery object if possible. Maybe should have named it `customizePanelEl` though...